templates/types/streaming/express/package.json

```diff
@@ -27,7 +27,8 @@
     "@e2b/code-interpreter": "^0.0.5",
     "got": "^14.4.1",
     "@apidevtools/swagger-parser": "^10.1.0",
-    "formdata-node": "^6.0.3"
+    "formdata-node": "^6.0.3",
+    "marked": "^14.1.2"
```

Comment:
> this is needed for the document generator tool as far as I can see

Yes, i tried to add dependency dynamically but it had a TS type checking issue (because we copied all the tool folder to the app). This change to keep the same tool dependencies as we did for other tools, but may be, we can add these tool dependencies whenever a tool is selected or using agent/multi-agent template.

Reply:
yes i thought so, to solve this we will need to just copy selected tools - not the scope of this PR. thanks